libs/designsystem/data-table/src/sortable/sortable.component.spec.ts

```diff
@@ -0,0 +1,118 @@
+
+const getFontWeight = DesignTokenHelper.fontWeight;
+
+describe('TableSortableComponent', () => {
+  let spectator: Spectator<TableSortableComponent>;
+
+  const createHost = createHostFactory({
+    component: TableSortableComponent,
+    declarations: [MockComponent(IconComponent)],
+  });
+
+  beforeEach(() => {
+    spectator = createHost(`
+    <table class="kirby-table">
+      <th [sortable]="true">Data1</th>
+    </table>
+    `);
+  });
+
+  describe('by default', () => {
+    it('should create', () => {
+      expect(spectator.component).toBeTruthy();
+    });
+
+    it('should hide the sorting icon', () => {
+      const icon = spectator.query('kirby-icon');
+
+      expect(icon).toHaveComputedStyle({
+        visibility: 'hidden',
+      });
+    });
+  });
+
+  describe('sortable', () => {
+    it('should render a button, when sortable is true', () => {
+      const button = spectator.query('button');
+
+      expect(button).toBeTruthy();
+    });
+
+    it('should render a text, when sortable is false', () => {
+      spectator.setInput('sortable', false);
+
+      expect(spectator.element.firstChild.nodeType).toBe(Node.TEXT_NODE);
+    });
+  });
+
+  describe('sortDirection', () => {
+    it('should render an "arrow-up" icon, when sortDirection is "asc"', () => {
+      spectator.setInput('sortDirection', 'asc');
+
+      const icon = spectator.query('kirby-icon');
+
+      expect(icon.attributes.getNamedItem('ng-reflect-name').value).toBe('arrow-up-fill');
+    });
+
+    it('should render an "arrow-down" icon, when sortDirection is "desc"', () => {
+      spectator.setInput('sortDirection', 'desc');
+
+      const icon = spectator.query('kirby-icon');
+
+      expect(icon.attributes.getNamedItem('ng-reflect-name').value).toBe('arrow-down-fill');
+    });
+  });
+
+  describe('when active', () => {
+    it('should get the "active" class when the active input is set to true', () => {
+      spectator.setInput('active', true);
+
+      expect(spectator.element).toHaveClass('active');
+    });
+
+    it('should make the text bold', () => {
+      spectator.setInput('active', true);
+
+      expect(spectator.element).toHaveComputedStyle({
+        'font-weight': getFontWeight('bold'),
+      });
+    });
+  });
+
+  describe('align', () => {
+    it('should apply "justify-content: flex-start" if align is set to "start"', () => {
+      spectator.setInput('alignment', 'start');
+
+      const button = spectator.query('button');
+
+      expect(button).toHaveComputedStyle({
+        'justify-content': 'flex-start',
+      });
+    });
+
+    it('should apply "justify-content: center" if align is set to "start"', () => {
+      spectator.setInput('alignment', 'center');
+
+      const button = spectator.query('button');
+
+      expect(button).toHaveComputedStyle({
+        'justify-content': 'center',
+      });
+    });
+
+    it('should apply "justify-content: flex-end" if align is set to "start"', () => {
```

Comment:
Consider simplifying descriptions as the tests themselves holds the details - e.g.:
`it('should left-align content when align is set to "start"', () => {` etc.